tests/virtctl/ssh.go

```diff
@@ -0,0 +1,86 @@
+package virtctl
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"golang.org/x/crypto/ssh"
+
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/clientcmd"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/libvmi"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+var _ = Describe("[sig-compute][virtctl]SSH", func() {
+	var pub ssh.PublicKey
+	var keyFile string
+	var virtClient kubecli.KubevirtClient
+
+	BeforeEach(func() {
+		var err error
+		virtClient, err = kubecli.GetKubevirtClient()
+		Expect(err).ToNot(HaveOccurred())
+		// Disable SSH_AGENT to not influence test results
+		Expect(os.Setenv("SSH_AUTH_SOCK", "/dev/null")).To(Succeed())
```

Comment:
I think this needs a cleanup after the tests. Potentially it can cause side effects and affect execution of other tests. There is a nice feature for such cases: https://onsi.github.io/ginkgo/#cleaning-up-our-cleanup-code-defercleanup

Reply:
This follows the same mechanism as the scp tests. So, I'm not completely sure. @rmohr do you recall we you needed to set the variable?
For the local ssh we don't need as it needs to be set as part of the `exec.Comman`, but I think it is necessary for the ssh native method

Reply:
I think we could also set it on a more global level. I want to avoid that a local running ssh agent can influence the outcome. Like it can happen that there are 10 keys registered in the ssh agent. They could all be tried before the generated key is used. At this point we could already be rejected from the server due to too many autonetication errors.

i can't think of any integration test at the moment where we would rely without special setup (which would be taken care of for that test) on the ssh agent. It should imo be ok to just disable any influence it cloud have.